pom.xml

```diff
@@ -129,7 +133,7 @@
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-archiver</artifactId>
-      <version>4.2.2</version>
+      <version>4.2.5</version>
```

Comment:
4.2.7 is out

Reply:
> 4.2.7 is out

But until the maven-war-plugin uses Java 8, that would require to downgrade commons-compress and plexus-utils, and my guess is that to simplify transitive dependency management, this just sets the latest that has bytecode 51.

Reply:
We are moving everything to 8

Reply:
Sure, yet this project is still Java 7:
https://github.com/apache/maven-war-plugin/blob/4355af5a697de7c1a782da8aed6f0b45993bf384/pom.xml#L68

So, ideally bump here to Java 8 and update plexus-archiver to 4.2.7, or bump it on another PR.

Reply:
PR opened https://github.com/apache/maven-war-plugin/pull/23